app/models/webhook.rb

```diff
@@ -7,6 +7,8 @@
 class Webhook < ApplicationRecord
   validates :params, presence: true
 
+  scope :pending_notification, -> { where(notified_at: nil) }
```

Comment:
I _think_ this is fine, but my first instinct on seeing it in action was to misread it as a relation or list of attributes - `Webhook.pending_notifications`, probably just me being tired!